CONTRIBUTING.md

```diff
@@ -51,6 +51,65 @@ implementation of the issue, ask them in the issue instead of the PR.
 
 The following instructions are for developers and contributors to cuDF OSS development. These instructions are tested on Linux Ubuntu 16.04 & 18.04. Use these instructions to build cuDF from source and contribute to its development.  Other operating systems may be compatible, but are not currently tested.
 
+### Code Formatting
+
+#### Python
+
+cuDF uses [Black](https://black.readthedocs.io/en/stable/),
+[isort](https://readthedocs.org/projects/isort/), and
+[flake8](http://flake8.pycqa.org/en/latest/) to ensure a consistent code format
+throughout the project. `Black`, `isort`, and `flake8` can be installed with
```

Comment:
It would be good to add these as dependencies to the [cudf_dev conda envs](https://github.com/rapidsai/cudf/blob/branch-0.9/conda/environments/cudf_dev_cuda10.0.yml)